Ryujinx.Memory/MemoryManagerUnixHelper.cs

```diff
@@ -151,10 +151,9 @@ private static int MmapFlagsToSystemFlags(MmapFlags flags)
                 }
             }
 
-            if (OperatingSystem.IsMacOSVersionAtLeast(10, 14))
+            if (flags.HasFlag(MmapFlags.MAP_JIT_DARWIN) && OperatingSystem.IsMacOS())
```

Comment:
Considering other path checks version, might be worth keeping it that way.

```suggestion
            if (flags.HasFlag(MmapFlags.MAP_JIT_DARWIN) && OperatingSystem.IsMacOSVersionAtLeast(10, 14))
```